lib/config/presets/internal/replacements.ts

```diff
@@ -51,6 +52,18 @@ export const presets: Record<string, Preset> = {
     ],
     ignoreDeps: [], // Hack to improve onboarding PR description
   },
+  'airbnb-prop-types-to-prop-types-tools': {
+    '`airbnb-prop-types` was given to a new maintainer and renamed',
+    packageRules: [
+      {
+        matchCurrentVersion: '^2',
+        matchDatasources: ['npm'],
+        matchPackageNames: ['airbnb-prop-types'],
+        replacementName: 'prop-types-tools',
+        replacementVersion: '^2.17.0',
```

Comment:
I think this needs to be a version and not range?

Reply:
Yes that is correct

Reply:
ah ok, but if the latest is 2.17.1 or 2.18.0 i'd want that to be what's used :-/ why would the replacement version need to be pegged?

Reply:
or do i have to come back here and make a PR every time i publish a v2 update to prop-types-tools?

Reply:
I think this is the sequence that happens:

1. At the time of merging the replacement preset `2.17.0` is latest
2. @ljharb releases new version, let's say `2.50.0`
3. Renovate end-users gets the replacement PR to the pinnede `2.17.0` version, and merges it.
4. Renovate creates a PR to update to the latest version `2.50.0`, when it meets the other requirements (passed the `minimumReleaseAge` days, for example)
5. @ljharb releases new major `v3` version
6. Renovate creates a update PR for the new major

So you do _not_ need to update the "replacement preset" after you're done with this PR. :wink:

Reply:
The idea is that, the replacement is identical to the last version under the old naming schema. 

Later upgrades of dependencies are than provided separatly, after an user merges this. 

The reasoning is that way you should not get any CI failures which are actually reasoned on actual code changes, but rather only references to the old package. This makes it easier and more predictable changing packages.

Reply:
sure. but semver means that they won't get CI failures from any v2 upgrade :-)

Reply:
That would be the ideal scenario, though that is sadly often not the case.